test/checks/aria/required-children.js

```diff
@@ -49,6 +49,12 @@ describe('aria-required-children', function () {
 		assert.isTrue(checks['aria-required-children'].evaluate.call(checkContext, node));
 	});
 
+	it('should pass a native input with role comboxbox when missing child is role textbox', function () {
+		fixture.innerHTML = '<input type="text" role="combobox" aria-owns="listbox" id="target"></div><p role="listbox" id="listbox">Nothing here.</p>';
```

Comment:
It's cool! The old method was still needed for develop-2x. Hopefully we'll be able to kill off that branch before too long.